database/migrations/2023_04_11_000008_create_go_load_up_cart_requirements_table.php

```diff
@@ -8,16 +8,18 @@
 {
     public function up()
     {
-        Schema::create('go_load_up_cart_requirements', function (Blueprint $table) {
-            $table->id();
-            $table->unsignedInteger('service_product_variant_id')->unique();
-            $table->json('first_set_of_required_shopify_product_ids')->nullable();
-            $table->json('second_set_of_required_shopify_product_ids')->nullable();
-            $table->text('error_message');
-            $table->timestamps();
+        if(!Schema::hasTable('go_load_up_cart_requirements')) {
```

Comment:
Curious why are you using `!Schema::hasTable` migrations should only be ran once, so not sure if this would ever be false?

Reply:
@OniiCoder check this comment

Reply:
ooooooooo can you elaborate a bit @bmmage ? So is the implication here that `if(!Schema::hasTable` may never hit because there would never be a case where this migration would ever be ran outside of this initial execution? So if there was a hypothetically "new" migration or purge, the table would not exist so then why have this.

In my mind, this feels more like a safety in the event that there is a freak accident where the table happened to have existed and we didnt want to override its established settings. Could that ever happen?

Reply:
@bmmage thanks for the catch, I was testing something. Fixed!

I noticed that whenever you run lego:install, it shows that tables already exist, I added the check to avoid that error to update the package successfully and forgot to remove it.